### ISSUE TITLE
Name is missing for "Roll up entity and Hierarchy route" , and control type is provided as "Group"

### ISSUE THREAD

**Issue by fsteffi** [OfficeDev/office-ui-fabric-react#11730]
Environment Details:​​
Browser Versions: Microsoft Edge 44.18362.387.0,
Chrome Version 77.0.3865.120 (Official Build) (64-bit)
Windows Version: Windows10​​​
NVDA 2019.2.1

Repro Steps:​​
1. Launch the application URL
2. Select application as "Sales hub" from the list of application.
3. From the change area which is present in the bottom select "App Settings"​
4. Navigate to "Site Map" and then Navigate to "Goal Management" and select "Forecast Configuration"​
5. Fast Track Your Forecast Screen will appears.​
6. Click on "Org chart forecast"​
7. Forecast Configuration screen will appears​
8. Open Accessibility insights for desktop and hover over the control.

Actual:​
Name is missing for "Roll up entity and Hierarchy route" , and control type is provided as "Group" which is not correct.

Same issue is observed with NVDA.

Same issue for the following page:
1.Forecast configuration-(name should be field and control as combo-box.)
2.Configure -(control for the dropdown under Hierarchy is set as group and edit under name has group set for it as well as information icon)

Expected:​​
Name should be provided as ""Roll up entity" and Hierarchy route".
Role should be provided as combo box.

User Impact:​
Screen reader user will not be able to know proper information about the control.

**Comment by xugao** [comment]
@fsteffi - pls provide detailed repro steps using example inside https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown

**Comment by fsteffi** [comment]
@xugao Hit the url : https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown
and try to navigate to "Disabled example with defaultSelectedKey" field.


Actual:​
Name is missing for "Disabled example with defaultSelectedKey" , and control type is provided as "Group" which is not correct.

Expected:​​
Name should be provided as "Disabled example with defaultSelectedKey", Role should be provided as combo box.

**Comment by xugao** [comment]
@fsteffi - this should be fixed in our latest version https://fabricweb.z5.web.core.windows.net/oufr/7.83.1/#/examples/dropdown by recent change #11678. pls check again
